### PR TITLE
Feature/static code analysis

### PR DESCRIPTION
**Pull request by NikosNe**
Addresses #22

### CODE REVIEW

**Comment by axthosarouris** [comment]
Approved if the conflicts are resolved

**Comment by NikosNe** [comment]
We have a "it works on my machine" case. Are you getting the same error as the one in the check?

**Comment by axthosarouris** [comment]
> We have a "it works on my machine" case. Are you getting the same error as the one in the check?

It works on me as well

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: PracticeWebApp/Startup.cs [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
+                {
+                    app.UseDeveloperExceptionPage();
+                    app.UseSwagger();
+                    app.UseSwaggerUI(options => { SwaggerPageAppearsAtProjectRoot(options); });
+                    void SwaggerPageAppearsAtProjectRoot(SwaggerUIOptions swaggerUiOptions)
+                    {
+                        swaggerUiOptions.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+                        swaggerUiOptions.RoutePrefix = string.Empty;
+                    }
+                }
```
[the remark this answers is not in the document]
Comment (axthosarouris) [reply PracticeWebApp/Startup.cs:52]: I have also created a PR to your PR, just to make sure that the tests run :)

**Pull request merged**